### PR TITLE
Updated etherscan plugin to include Optimism Sepolia chainId

### PR DESCRIPTION
**Pull request by FaisalAli19**
## Description

Added the Optimism Sepolia chain id [11155420] to the `apiUrls` object in the etherscan plugin file. Also included is the corresponding API URL 'https://api-sepolia-optimistic.etherscan.io/api'.

Before submitting this issue, please make sure you do the following.

- [x] Read the [contributing guide](https://wagmi.sh/dev/contributing)
- [x] Added or updated tests (and snapshots) related to the changes made.

### CODE REVIEW

File: packages/cli/src/plugins/etherscan.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ const apiUrls = {
   // Optimism
   [10]: 'https://api-optimistic.etherscan.io/api',
   [420]: 'https://api-goerli-optimistic.etherscan.io/api',
+  [11155420]: 'https://api-sepolia-optimistic.etherscan.io/api',
```
Comment (tmm) [inline packages/cli/src/plugins/etherscan.ts:13]:
```suggestion
  [11_155_420]: 'https://api-sepolia-optimistic.etherscan.io/api',
```

**Review by tmm — CHANGES_REQUESTED** [review]
Please [add a changeset](https://wagmi.sh/dev/contributing#_9-versioning)

File: packages/cli/src/plugins/etherscan.ts (hunk 1 above)
Comment (FaisalAli19) [reply packages/cli/src/plugins/etherscan.ts]: Formatted the chainId to the above mentioned format.

**Comment by FaisalAli19** [comment]
Add the changeset please check when you have time. Thanks

File: .changeset/real-sheep-begin.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@wagmi/cli": minor
```
Comment (tmm) [inline .changeset/real-sheep-begin.md:2]:
```suggestion
"@wagmi/cli": patch
```

File: .changeset/real-sheep-begin.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+Added the Optimism Sepolia chain id [11_155_420] to the apiUrls object in the etherscan plugin file. Also included is the corresponding API URL 'https://api-sepolia-optimistic.etherscan.io/api'.
```
Comment (tmm) [inline .changeset/real-sheep-begin.md:5]:
```suggestion
Added Optimism Sepolia Etherscan support
```

**Pull request merged**